app/components/key_pair_generator_component.js

```diff
@@ -0,0 +1,40 @@
+import { trackEvent } from '@18f/identity-analytics';
+
+class KeyPairGeneratorElement extends HTMLElement {
+  connectedCallback() {
+    this.generateKeyPair().then(() => {
+      if (this.duration === undefined) {
+        this.duration = -1;
+      }
+      trackEvent('IdV: key pair generation', {
+        duration: this.duration.toFixed(0),
```

Comment:
If we wanted to keep this as a numeric type (e.g.. unsure of implications of using a string in CloudWatch aggregation queries), could we round it to drop the decimal value?

```suggestion
        duration: Math.round(this.duration),
```